src/libs/reportUtils.js

```diff
@@ -226,8 +225,7 @@ function canShowReportRecipientLocalTime(personalDetails, myPersonalDetails, rep
         && reportRecipient
         && reportRecipientTimezone
         && currentUserTimezone.selected
```

Comment:
Do we need this anymore?

Reply:
We can also remove this ONYXKEYS.MY_PERSONAL_DETAILS is not needed.

Reply:
We need those to ensure `reportRecipient` && `reportRecipientTimezone` are available to show the time. The result of this also used to change styles.

<img width="694" alt="Screenshot 2022-02-25 at 12 56 19 PM" src="https://user-images.githubusercontent.com/85645967/155672927-4543f899-b3e1-4c9c-b622-f32f93f2a034.png">

`currentUserTimezone.selected` can be removed.

What do you say?

Reply:
So you should remove it. Please clean up extra things.

Reply:
Need to remove `currentUserTimezone.selected` this alone right? @parasharrajat

Reply:
You said this can be removed so please remove it. And try to see what can be cleaned up in the code related to this.  I think this ONYXKEYS.MY_PERSONAL_DETAILS is also unused now.

Reply:
Yes will remove the unused code related to those!